### ISSUE TITLE
[A11y_Bug]: Accessibility checker works as chrome extension but not as a npm package for certain URLs

### ISSUE THREAD

**Issue by ShrutiP26** [IBMa/equal-access#1208]
### Project

a11y checker

### Browser

Chrome

### Operating System

Windows

### Automated testing tool and ruleset

IBM Access Accessibility Checker - latest deployment

### Assistive technology

No

### Description

For example we are able to generate accessibility report for https://www.alleanza.it/risparmio/ using the chrome extension
<img width="949" alt="image" src="https://user-images.githubusercontent.com/72301443/208909154-de9ca23c-34dc-43ef-8d3e-84cef9335a04.png">

But if we do the same using the below npx command (accessibility-checker version - 3.1.41) it does not generate any report
npx achecker --outputFolder Workspace\Reports --outputFormat html "https://www.alleanza.it/risparmio/"

Is this a bug, can you please assist with this issue?

### Steps to reproduce

Use below npx command (accessibility-checker version - 3.1.41)
npx achecker --outputFolder Workspace\Reports --outputFormat html "https://www.alleanza.it/risparmio/"

**Comment by ShrutiP26** [comment]
I tried without the quotes it is not working and yes I am using windows system. I am running this command in visual studio code with "accessibility-checker": "^3.1.41" package in dependencies.

**Comment by aliunwala** [comment]
Could you shoot a screen capture of what happens? Is there any logging that gets output to the terminal screen?

**Comment by aliunwala** [comment]
From triage meeting we do see the scan from the exension we are getting results: 
![image](https://user-images.githubusercontent.com/4451128/216450624-8c3884b7-c907-4493-bd2b-df1d6e5f8176.png)

But we are seeing a hang (on windows and mac) when we try to run 
```
npx achecker --outputFolder .  --outputFormat html https://www.alleanza.it/risparmio/ 
```
moving this to backlog to try and debug this in a future sprint.

**Comment by tombrunet** [comment]
@ShrutiP26 The reason this happens is because the page server sets a content security policy that prevents our engine script from loading. https://github.com/IBMa/equal-access/pull/1485 will provide a better fail scenario to indicate his for future releases.

**Comment by tombrunet** [comment]
As an example, when that's deployed, it will says:
> Error: Evaluation failed: Error: Unable to load engine into https://www.alleanza.it/risparmio/. This can happen if the page server sets a Content-Security-Policy that prevents https://cdn.jsdelivr.net/npm/accessibility-checker-engine@3.1.50/ace.js from loading.

**Comment by tombrunet** [comment]
#1507 May provide a more complete fix in directly loading the engine rather than through script. This would avoid CSP errors.

**Comment by ErickRenteria** [comment]
merged
